server/models/Collective.js

```diff
@@ -225,6 +225,14 @@ export default function(Sequelize, DataTypes) {
 
       address: DataTypes.STRING,
 
+      countryISO: {
+        type: DataTypes.STRING,
+        validate: {
+          len: 2,
+          isISO31661Alpha2: true,
```

Comment:
I can't find this one in the docs, any pointer to how it works?

Reply:
😆  Omg! Added this late last night, sequelize uses [validator](https://www.npmjs.com/package/validator) internally which has  `isISO31661Alpha2()` implemented I interestingly added it hoping to copy the [function](https://github.com/chriso/validator.js/blob/master/lib/isISO31661Alpha2.js) to a `util` file since it wouldn't make sense to install a new a package just to verify a single string.  I tested it and somehow worked. Good sleep is good lol, thought I copied it already. I will do just that asap.

Reply:
I'm personally ok if you want to add `validator` as a dependency, it's a good library that we could use for many other validations. Also you want to keep this kind of things up-to-date: country codes can appear and disappear, it's good to rely on something that updates it  for us :slightly_smiling_face: 

Just be careful, we use fixed dependencies versions. You can do `npm install validator --save-exact` to achieve that.